uobtheatre/bookings/test/test_models.py

```diff
@@ -904,3 +904,24 @@ def test_booking_pay_integration():
     assert isinstance(payment.provider_payment_id, str)
     assert payment.provider == Payment.PaymentProvider.SQUARE_ONLINE
     assert payment.type, Payment.PaymentType.PURCHASE
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "initialState, finalState",
+    [(True, True), (False, True)],
+)
+def test_ticket_check_in(initialState, finalState):
+    """
+    Test ticket check in method
+    """
+
+    booking = BookingFactory()
+    psg = PerformanceSeatingFactory(performance=booking.performance)
+    ticket = TicketFactory(
+        booking=booking, seat_group=psg.seat_group, checked_in=initialState
+    )
```

Comment:
Do we need these two factories? Or can we just do
```suggestion
    ticket = TicketFactory(
        checked_in=initialState
    )
```